persistence/block.go

```diff
@@ -55,15 +76,51 @@ func (p PostgresContext) TransactionExists(transactionHash string) (bool, error)
 	return true, err
 }
 
-func (p PostgresContext) StoreTransaction(txResult modules.TxResult) error {
-	return p.txIndexer.Index(txResult)
+func (p PostgresContext) StoreTransactions() error {
```

Comment:
I think `IndexProposalTransactions` would be more representative of what this does.

Reply:
See comment above about ApplyBlock being used during state sync

Reply:
How about just `IndexTransactions`? Storing could be used with the merkle tree separately

Reply:
That makes sense.